Fix documentation section (#10031)
bokeh/bokeh: sphinx/source/docs/dev_guide/documentation.rst

    BOKEH_DOCS_CDN=local make clean html serve

> Built docs require to setup the ``GOOGLE_API_KEY`` environment variable for geo maps.

Reviewer: ```suggestion
Building the docs also requires setting the ``GOOGLE_API_KEY`` environment variable that `gmap` plots use.
```

> You can create a new API `create a new API Key <https://developers.google.com/maps/documentation/javascript/get-api-key>`_.

Reviewer: ```suggestion
You can `create a new API Key <https://developers.google.com/maps/documentation/javascript/get-api-key>`_.
```

Author: I agree, I've just deleted repeated phrase

> Or if you don't mind of having broken maps, you can setup the variable with a fake value before calling ``make``:

Reviewer: ```suggestion
Or, if you don't care whether `gmap` plots work, you can set the variable with a fake value before calling ``make``:
```
